### ISSUE TITLE
A couple of spelling errors in the docs

### ISSUE THREAD

**Issue by ukch** [pyeve/cerberus#390]
#### Support request / Bug report

When building the docs for distribution, I have run them through an automated spell-checker and come up with the following issues:

```
particulary :: particularly
allows to :: allows one to (*3)
occurance :: occurrence
intially :: initially
keywork :: keyword
superflous :: superfluous
lenght :: length
occured :: occurred
```
(In the table above, to the left is the actual spelling and to the right the expected spelling. `(*3)` means the issue occurs three separate times.)

Please consider correcting the spelling in the source files.

**Comment by funkyfuture** [comment]
thanks for your efforts. as an open source community project we rely on patches from the communities.

**Comment by nicolaiarocci** [comment]
Hello, thanks for the report. Would you mind submitting a pull request to fix them? We welcome first time contributors!

**Comment by funkyfuture** [comment]
Resolved with #401.
